### PR TITLE
Added Olombelo Ricky

### PR DESCRIPTION
**Pull request by Tsikitsiky**
Hi, 
This is my last PR for today, please have a look.

Thank you in advance,
Sarah

### CODE REVIEW

File: o/olombelo-ricky.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Olombelo Ricky
+
+![Olombelo Ricky](olombelo.jpg)
```
Comment (ndorfin) [inline o/olombelo-ricky.md]: Please make the image filename match the artist filename.

File: o/olombelo-ricky.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+Olombelona Ricky, also known as Ricky, is a Malagasy vocalist and roots musician based in Antananarivo, Madagascar. Launching his musical career in 1991, Ricky recorded his first album in 1998, sparking the phenomenon of "Rickymania". His socio-political messages and uncompromising approach to his career have earned him a strong following among urban youth within Madagascar. He is esteemed as a musical and cultural ethnologist by many of his fellow Malagasy musicians for his effort to capture and reinvent the traditional musical heritage of the island and that of the near-mythical Vazimba people in particular, believed by many Malagasy to be the island's earliest inhabitants. Despite an uncompromising approach to his career which slowed his international exposure, Ricky has performed at numerous international music festivals over the course of his career and regularly tours at home and abroad.
+
+**Genre:** Jazz
+
+**Artist homepage:** [OLombelo Ricky](https://www.last.fm/fr/music/Olombelo+Ricky/+wiki)
```
Comment (ndorfin) [inline o/olombelo-ricky.md]: Fix the capitalisation of the `L`

**Review by ndorfin — CHANGES_REQUESTED** [review]
This looks great, just a few more fixes to go!

File: o/olombelo-ricky.md (hunk 2 above)
Comment (Tsikitsiky) [reply o/olombelo-ricky.md]: I think I've made the change, please check.

**Review by ndorfin — APPROVED** [review]
Good work!

**Pull request merged**